Double quote to prevent globbing and word splitting.

[out script](https://github.com/homedepot/newrelic-deploy-resource/blob/master/out)

Lines 12, 14, 16, 18, 20, 25, 32, 33, 34, 55, 59, 62
[SC2086](https://github.com/koalaman/shellcheck/wiki/SC2086): Double quote to prevent globbing and word splitting.

Source: [ShellCheck](https://www.shellcheck.net/)

--- a/out
+++ b/out
@@ -9,26 +9,26 @@ PATH=/usr/local/bin:$PATH
 
 inputs=$(mktemp /tmp/resource-in.XXXXXX)
 
-cat > $inputs <&0
+cat > "$inputs" <&0
 
-app_id=$(jq -r '.source.app_id' < $inputs)
+app_id=$(jq -r '.source.app_id' < "$inputs")
 
-api_key=$(jq -r '.source.api_key' < $inputs)
+api_key=$(jq -r '.source.api_key' < "$inputs")
 
-json_file=$(jq -r '.params.json_file // ""' < $inputs)
+json_file=$(jq -r '.params.json_file // ""' < "$inputs")
 
-user=$(jq -r '(.params.user // empty)' < $inputs)
+user=$(jq -r '(.params.user // empty)' < "$inputs")
 
 
 if [ -n "${json_file}" ] && [ -f "${json_file}" ]; then
-  if [ "$(jq 'has("revision")' $json_file)" = "false" ]; then
+  if [ "$(jq 'has("revision")' "$json_file")" = "false" ]; then
     echo "revision is missing from the json_file. cannot continue"
     exit 1
   fi
 
-  revision=$(jq -r '.revision' $json_file)
-  changelog=$(jq -r '(.changelog // empty)' $json_file)
-  description=$(jq -r '(.description // empty)' $json_file)
+  revision=$(jq -r '.revision' "$json_file")
+  changelog=$(jq -r '(.changelog // empty)' "$json_file")
+  description=$(jq -r '(.description // empty)' "$json_file")
 fi
 
 body=$(jq -c -n \
@@ -49,14 +49,14 @@ body=$(jq -c -n \
 RESPONSE=$(curl -X POST "https://api.newrelic.com/v2/applications/${app_id}/deployments.json" \
      -H "X-Api-Key:${api_key}" -i \
      -H "Content-Type: application/json" \
-     -d ${body})
+     -d "${body}")
 
 
 
-STATUS=$(echo $RESPONSE | grep HTTP |  awk '{print $2}')
+STATUS=$(echo "$RESPONSE" | grep HTTP |  awk '{print $2}')
 
 if [ "$STATUS" -ne "200" ] && [ "$STATUS" -ne "201" ]; then
-  echo $RESPONSE
+  echo "$RESPONSE"
   exit 1;
 fi
 

